core/src/main/java/io/questdb/mp/WorkerPoolUtils.java

```diff
@@ -39,11 +40,25 @@
 import io.questdb.griffin.engine.groupby.GroupByMergeShardJob;
 import io.questdb.griffin.engine.groupby.vect.GroupByVectorAggregateJob;
 import io.questdb.griffin.engine.table.LatestByAllIndexedJob;
+import io.questdb.std.Files;
+import io.questdb.std.Os;
 import io.questdb.std.Rnd;
 import io.questdb.std.datetime.Clock;
 
 public class WorkerPoolUtils {
 
+    public static void setupAsyncMunmapJob(WorkerPool pool, CairoEngine engine) {
+        CairoConfiguration config = engine.getConfiguration();
+        if (config.getAsyncMunmapEnabled()) {
+            assert Os.isPosix();
```

Comment:
could you please elaborate on this assert? How's test passing on windows? There seem to be no guards against setting this job up on windows